sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 0.1.0-preview.11 (Unreleased)
+
+
 ## 0.1.0-preview.10 (2021-03-09)
```

Comment:
@chidozieononiwu why this we not block the release for this changelog? I thought we fail if the change log entry is empty when trying to do the release.